merfi/backends/rpm_sign.py

```diff
@@ -57,18 +60,19 @@ def sign(self):
             raise RuntimeError('specify a --key for signing')
         logger.info('Starting path collection, looking for files to sign')
         repos = RepoCollector(self.path)
```

Comment:
true, but in this case this class is only used on `rpm-sign` sub-commands. No? There will never be a need to look for deb files